src/helpers/utils.ts

```diff
@@ -48,3 +48,11 @@ export async function getEmailAddresses(email: string): Promise<{ address: strin
 export async function getUniqueEmails(): Promise<{ email: string }[]> {
   return await db.queryAsync('SELECT DISTINCT email FROM subscribers WHERE verified > 0');
 }
+
+export function isValidEmail(input: string) {
```

Comment:
what do you think about covering those functions with unit tests?

Reply:
Pushed on #49 by mistake